### ISSUE TITLE
laravel horizon jobs stuck in pending

### ISSUE THREAD

**Issue by khalidgxg** [laravel/horizon#1280]
### Horizon Version

5.15

### Laravel Version

10.09.0

### PHP Version

8.1

### Redis Driver

Predis

### Redis Version

2.1.2

### Description

Hi all,

I am writing to report a bug related to the job processing in Laravel Horizon. Specifically, I have encountered a situation where a job appears to be **stuck** in the "**pending**" status despite having a "**completed_at**" timestamp, and another job is stuck in the "**reserved**" status without progressing further.

Here are the details of the problematic jobs:

- ### Job with "pending" status but "completed_at" :

  "**status**": "pending",
 "**completed_at**": "1685113859.5327",
 "**reserved_at**": "1685113859.5284",
 

- ### Job with "reserved" status :

  "**status**": "reserved",
 "**completed_at**": null,
 "**reserved_at**": "1685110558.3953",

In the **first** case, the job has a "**completed_at**" timestamp indicating its successful completion, but it **remains** in the "**pending**" status. On the other hand, the second job is stuck in the "**reserved**" status without progressing further.

Could you please investigate this issue and provide guidance on how to resolve it? It seems that the job statuses are not being updated correctly, causing confusion in monitoring and processing.

Thank you for your attention to this matter. I look forward to your response and assistance in resolving this bug.

Best regards,
khalid

### Steps To Reproduce

1. Set up a Laravel application with Laravel Horizon installed. Make sure you have the necessary dependencies and configurations in place.
2. 
3. Create a job that exhibits the issue. For example, you can create a custom job class StuckJob that performs some task, such as writing to a log file or making an API request.
4. 
5. Configure your application to use Laravel Horizon as the queue driver. Ensure that the queue connections and supervisors are properly set up.
6. 
7. Push multiple instances of the StuckJob to the queue using the Laravel Horizon job queueing mechanism. You can use the dispatch() function or Horizon-specific methods like Horizon::queue() to push the jobs.
8. 
9. Monitor the Horizon dashboard to observe the job processing. Keep an eye on the status of the jobs you pushed.
10. 
11. Check if any of the jobs get stuck in the "pending" status despite having a "completed_at" timestamp. Note down the relevant job ID, connection, queue, and payload details.
12. 
13. Repeat the process with another job to observe if any jobs get stuck in the "reserved" status without progressing further. Again, note down the relevant job ID, connection, queue, and payload details.
14. 
15. Take note of the Laravel Horizon version you are using in your application.

**Comment by driesvints** [comment]
Hey there,

Can you first please try one of the support channels below? If you can actually identify this as a bug, feel free to open up a new issue with a link to the original one and we'll gladly help you out.

- [Laracasts Forums](https://laracasts.com/discuss)
- [Laravel.io Forums](https://laravel.io/forum)
- [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
- [Discord](https://discordapp.com/invite/KxwQuKb)
- [Larachat](https://larachat.co)
- [IRC](https://web.libera.chat/?nick=laravelnewbie&channels=#laravel)

Thanks!

**Comment by khalidgxg** [comment]
ok thanks

**Comment by SimonMacIntyre** [comment]
I'm reproducing this 100% of the time

**Comment by fabriciojs** [comment]
Got the same thing happening, latest Laravel & Horizon versions.

Jobs even though processed correctly keep accumulating on the Pending list in the Dashboard, although the queues are empty.

I could not find more accurate reports or solutions specifically to this situation.

@khalidgxg did you learn what was causing it for you?

And @driesvints I assume if one can showcase a repo with Laravel+Horizon that could consistenly show/reproduce the issue happening as reported, it would then qualify for you guys to pursue it as a bug, right? I might try to put something together.

**Comment by cccdz** [comment]
@driesvints @fabriciojs https://github.com/laravel/horizon/issues/1034

**Comment by driesvints** [comment]
Hey all. This should have been fixed already in 4.x, does that not work for you? https://github.com/laravel/telescope/pull/1349

**Comment by driesvints** [comment]
Telescope that is.

**Comment by cccdz** [comment]
@driesvints https://github.com/laravel/horizon/issues/1185
This happens when the consumer executes the event faster than the production event, because horizon is implemented through events, which are first dropped into a queue and then triggered, so there may be a situation where the consumer finishes consuming before starting to execute the event.

**Comment by cccdz** [comment]
<img width="878" alt="image" src="https://github.com/laravel/horizon/assets/45749908/a061ed66-a9b3-4ccb-b712-817188a7a3f5">

**Comment by driesvints** [comment]
Thank you. We'd appreciate any help through a PR for this.

**Comment by cccdz** [comment]
<img width="980" alt="image" src="https://github.com/laravel/horizon/assets/45749908/2b83475d-f5cd-47a5-9f24-600d2447a297">
<img width="980" alt="image" src="https://github.com/laravel/horizon/assets/45749908/8fc07648-af0c-4dc6-9acd-961ae5e7fe04">
I have a suggestion, is that these two listening methods can be used Lua script, and then in the pushed method of the Lua script to determine whether the key already exists, if the key exists, then on behalf of the completion of the event has been carried out, it is not written to the pending ordered list of collections, just update the hash key, I do not know if this is feasible.

**Comment by joelvh** [comment]
@driesvints is there a way to release these reserved jobs to they can be processed again?

**Comment by driesvints** [comment]
I don't know sorry/

**Comment by joelvh** [comment]
@themsaid do you maybe know if it's possible to release these jobs that get stuck as `reserved` to be processed again? Thanks!

**Comment by pnlinh** [comment]
Hi, i'm using Laravel 5.6 (PHP 7.4), Horizon version 2.2.2. I have a strangely issue, the jobs was pickup but status forever is `pending`. But if I trigger job manually then it works. Does some one have similar issue?

<img width="921" alt="image" src="https://github.com/laravel/horizon/assets/26193890/1b4e522a-c7c1-4190-9b57-7d0ab9d7e242">

**Comment by graemlourens** [comment]
@pnlinh you are WAY out of date with laravel, horizon & php. Please update to most recent versions and test again. There is no sense in asking for help with such outdated versions.

**Comment by pnlinh** [comment]
> @pnlinh you are WAY out of date with laravel, horizon & php. Please update to most recent versions and test again. There is no sense in asking for help with such outdated versions.

Thanks for your suggestion but my project cannot upgrade now. I added delay value to jobs, it seems it works.

**Comment by driesvints** [comment]
@pnlinh please try to focus the discussion on supported Laravel/Horizon versions, thanks.

**Comment by fwilliamconceicao** [comment]
Even with updated versions I still have this issue.

**Comment by ithuis** [comment]
i set 'TELESCOPE_JOB_WATCHER' to false in config, and they all came flooding back into completed.

"Watchers\JobWatcher::class => env('TELESCOPE_JOB_WATCHER', false)"

mentioned   https://github.com/laravel/telescope/pull/1349#issuecomment-1645425830

**Comment by Kladislav** [comment]
same issue..

**Comment by driesvints** [comment]
Hey all. Extra messages that you're experiencing this issue aren't really helpful. Instead, please try posting extra findings around the issue or help out with a PR, thanks.

**Comment by danmatthews** [comment]
@driesvints i'm having this too and it's deep rooted enough that I wonder if we could get some core team time to look into this? I'm happy to jump on a call with anyone to go through what's happening, and i'm currently recording a screencast to send to show the problem, let me know where I can send it if it's useful.

**Comment by lucaspanik** [comment]
> <img alt="image" width="980" src="https://private-user-images.githubusercontent.com/45749908/264977475-2b83475d-f5cd-47a5-9f24-600d2447a297.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg5NjMxODMsIm5iZiI6MTcwODk2Mjg4MywicGF0aCI6Ii80NTc0OTkwOC8yNjQ5Nzc0NzUtMmI4MzQ3NWQtZjVjZC00N2E1LTlmMjQtNjAwZDI0NDdhMjk3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjI2VDE1NTQ0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNkZTUyYjZjNmUxMTg4ZmRkNGY1ZTU0Y2RkMzZlYzY4MGQ0OGNiNDIzMTcyMDBmMmVkNTZmNjg4NzRkYzg0OGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CJEeInv7pjgBOzjV_aH9atAbZ7lhLOQAUeJgqEVf5Os"> <img alt="image" width="1124" src="https://private-user-images.githubusercontent.com/45749908/264982953-aad39b9e-c567-4933-a140-f290fdfc9399.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg5NjMxODMsIm5iZiI6MTcwODk2Mjg4MywicGF0aCI6Ii80NTc0OTkwOC8yNjQ5ODI5NTMtYWFkMzliOWUtYzU2Ny00OTMzLWExNDAtZjI5MGZkZmM5Mzk5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjI2VDE1NTQ0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThiYTg2YzMzOTE1NmQzNzkwNmY3ZGJlNjRjYWQwYTdiYmI3YjI1NTE3MmQ5MjZiZjcwOTliZjdmM2YwYzQ0NDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.IwURnJnGTih3aEIDbMOklI0FdXPcYnQn5cp4QvBjr8M"> I have a suggestion, is that these two listening methods can be used Lua script, and then in the pushed method of the Lua script to determine whether the key already exists, if the key exists, then on behalf of the completion of the event has been carried out, it is not written to the pending ordered list of collections, just update the hash key, I do not know if this is feasible.

I've been facing this problem for almost 3 years, where the internal solution provided is sleep(3) inside all jobs. 
https://github.com/laravel/horizon/issues/1034

What does the answer above make sense, since with sleep(3) the events have time to orchestrate themselves normally.

Would this be a possible point of investigation?

**Comment by fwilliamconceicao** [comment]
> > <img alt="image" width="980" src="https://private-user-images.githubusercontent.com/45749908/264977475-2b83475d-f5cd-47a5-9f24-600d2447a297.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg5NjMxODMsIm5iZiI6MTcwODk2Mjg4MywicGF0aCI6Ii80NTc0OTkwOC8yNjQ5Nzc0NzUtMmI4MzQ3NWQtZjVjZC00N2E1LTlmMjQtNjAwZDI0NDdhMjk3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjI2VDE1NTQ0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNkZTUyYjZjNmUxMTg4ZmRkNGY1ZTU0Y2RkMzZlYzY4MGQ0OGNiNDIzMTcyMDBmMmVkNTZmNjg4NzRkYzg0OGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CJEeInv7pjgBOzjV_aH9atAbZ7lhLOQAUeJgqEVf5Os"> <img alt="image" width="1124" src="https://private-user-images.githubusercontent.com/45749908/264982953-aad39b9e-c567-4933-a140-f290fdfc9399.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg5NjMxODMsIm5iZiI6MTcwODk2Mjg4MywicGF0aCI6Ii80NTc0OTkwOC8yNjQ5ODI5NTMtYWFkMzliOWUtYzU2Ny00OTMzLWExNDAtZjI5MGZkZmM5Mzk5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjI2VDE1NTQ0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThiYTg2YzMzOTE1NmQzNzkwNmY3ZGJlNjRjYWQwYTdiYmI3YjI1NTE3MmQ5MjZiZjcwOTliZjdmM2YwYzQ0NDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.IwURnJnGTih3aEIDbMOklI0FdXPcYnQn5cp4QvBjr8M"> I have a suggestion, is that these two listening methods can be used Lua script, and then in the pushed method of the Lua script to determine whether the key already exists, if the key exists, then on behalf of the completion of the event has been carried out, it is not written to the pending ordered list of collections, just update the hash key, I do not know if this is feasible.
> 
> I've been facing this problem for almost 3 years, where the internal solution provided is sleep(3) inside all jobs. #1034
> 
> What does the answer above make sense, since with sleep(3) the events have time to orchestrate themselves normally.
> 
> Would this be a possible point of investigation?

This worked for me for a couple of months, but since the application scaled up and we had more workload this became a huge headhache.

What I'm doing right now it's migrating everything for serverless services, jobs, and isolated applications with C#. 

The only way to stop this behavior is to stop using Horizon for huge workloads.

**Comment by cccdz** [comment]
```
class RedisJobRepository extends HorizonRedisJobRepository
{
    /**
     * 保留
     *
     * @param $connection
     * @param $queue
     * @param JobPayload $payload
     * @return void
     * @throws RedisException
     */
    public function reserved($connection, $queue, JobPayload $payload): void
    {
        // 循環總時長
        $totalTime = 0;

        // 如果horizon的任務不是pending狀態
        while ('pending' !== redis('horizon')->hget($payload->id(), 'status')) {
            // 如果循環時間大於等於1s
            if ($totalTime >= 1000000) {
                break;
            }

            // sleep 5ms
            usleep(5000);

            $totalTime += 5000;
        }

        parent::reserved($connection, $queue, $payload);
    }
}


$this->app->singleton(JobRepository::class, RedisJobRepository::class);
```

It can be temporarily avoided in this way.

**Comment by fwilliamconceicao** [comment]
> ```
> class RedisJobRepository extends HorizonRedisJobRepository
> {
>     /**
>      * 保留
>      *
>      * @param $connection
>      * @param $queue
>      * @param JobPayload $payload
>      * @return void
>      * @throws RedisException
>      */
>     public function reserved($connection, $queue, JobPayload $payload): void
>     {
>         // 循環總時長
>         $totalTime = 0;
> 
>         // 如果horizon的任務不是pending狀態
>         while ('pending' !== redis('horizon')->hget($payload->id(), 'status')) {
>             // 如果循環時間大於等於1s
>             if ($totalTime >= 1000000) {
>                 break;
>             }
> 
>             // sleep 5ms
>             usleep(5000);
> 
>             $totalTime += 5000;
>         }
> 
>         parent::reserved($connection, $queue, $payload);
>     }
> }
> 
> 
> $this->app->singleton(JobRepository::class, RedisJobRepository::class);
> ```
> 
> It can be temporarily avoided in this way.

This is a good solution tho. But have you tested with a huge workload? My workload's very big and when I started adding 2000 sleep everything started to overlap. I didn't try with 5k, might be it's a good workaround but anyway, it's not a good solution.

**Comment by cccdz** [comment]
> > ```
> > class RedisJobRepository extends HorizonRedisJobRepository
> > {
> >     /**
> >      * 保留
> >      *
> >      * @param $connection
> >      * @param $queue
> >      * @param JobPayload $payload
> >      * @return void
> >      * @throws RedisException
> >      */
> >     public function reserved($connection, $queue, JobPayload $payload): void
> >     {
> >         // 循環總時長
> >         $totalTime = 0;
> > 
> >         // 如果horizon的任務不是pending狀態
> >         while ('pending' !== redis('horizon')->hget($payload->id(), 'status')) {
> >             // 如果循環時間大於等於1s
> >             if ($totalTime >= 1000000) {
> >                 break;
> >             }
> > 
> >             // sleep 5ms
> >             usleep(5000);
> > 
> >             $totalTime += 5000;
> >         }
> > 
> >         parent::reserved($connection, $queue, $payload);
> >     }
> > }
> > 
> > 
> > $this->app->singleton(JobRepository::class, RedisJobRepository::class);
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 通过这种方式可以暂时避免。
> 
> 这是一个很好的解决方案。但你测试过巨大的工作量吗？我的工作量非常大，当我开始添加 2000 睡眠时，一切都开始重叠。我没有尝试使用 5k，这可能是一个很好的解决方法，但无论如何，这不是一个好的解决方案。

I this is within 1 second to detect whether it is a pending state, every 5ms cycle detection, if it is a pending state means that the event has been executed, you can carry out the next operation, I also do the anti-dumbness, if 1 second after the event has not been executed, the task will not care about it, so that he stays in the pending list, but this extreme case is almost zero!

**Comment by driesvints** [comment]
Hi all. This issue is now one year old. We haven't gotten any action on it any longer and nobody seems to have attempted a PR. There for we'll be closing this one. If anyone still finds a solution to this one we'd be more than willing to accept a PR. Thanks

**Comment by graemlourens** [comment]
Just for the record: the issue is happening for us nearly daily, even with a very low system baseload of 1 million jobs a day, whereby approximately 3-10 jobs still remain in pending, even if completed.

Still happens with most recent laravel and horizon version currently available.

**Comment by buglinjo** [comment]
Why isn't this issue still open? Still happening to everyone here and have not seen any clear solution in the comments.

**Comment by gavnyx** [comment]
![Image](https://github.com/user-attachments/assets/10d65eab-154f-48d3-8e18-db73695f93c0)

Stuck on pending too, I'm using Laravel 11 and "laravel/horizon": "^5.29"

**Comment by Parables** [comment]
At this point, I am switching to using Supervisor... The headaches with horizon is too much...
Forget all the bells and whistles that comes with horizon.

For anyone(including my future self) who comes along this, here is my horizon config before the switch:
```horizon.php
    /*
    |--------------------------------------------------------------------------
    | Queue Worker Configuration
    |--------------------------------------------------------------------------
    |
    | Here you may define the queue worker settings used by your application
    | in all environments. These supervisors and settings handle all your
    | queued jobs and will be provisioned by Horizon during deployment.
    |
    */

    'defaults' => [],

    'environments' => [
        '*' => [
            'express' => [
                'queue' => ['express'],
                'connection' => 'redis',
                'balance' => 'auto',
                'autoScalingStrategy' => 'size',
                'minProcesses' => 1,
                'maxProcesses' => 2,
                'maxTime' => 3600,
                'maxJobs' => 500,
                'balanceMaxShift' => 1,
                'balanceCooldown' => 3,
                'tries' => 1,
                'timeout' => 120,
                'nice' => 0,
            ],

            'default' => [
                'queue' => ['default'],
                'connection' => 'redis',
                'balance' => 'auto',
                'autoScalingStrategy' => 'size',
                'minProcesses' => 1,
                'maxProcesses' => 2,
                'maxTime' => 14400,
                'maxJobs' => 10,
                'balanceMaxShift' => 1,
                'balanceCooldown' => 3,
                'tries' => 1,
                'timeout' => 7200,
                'nice' => 0,
            ],
        ],
    ],
```

And here is my Supervisor config that I am switching to:
```
[program:my-app-express-workers]
process_name=%(program_name)s_%(process_num)02d
command=php /my-app/artisan queue:work redis --queue=express --tries=1 --timeout=300 --sleep=3 --daemon --max-time=3600 --max-jobs=20
autostart=true
autorestart=true
stopasgroup=true
killasgroup=true
user=caddy
numprocs=3
redirect_stderr=true
stdout_logfile=/my-app/storage/logs/express-queue-workers.log
stopwaitsecs=420

[program:my-app-default-workers]
process_name=%(program_name)s_%(process_num)02d
command=php /my-app/artisan queue:work redis --queue=default --tries=1 --timeout=7200 --sleep=3 --daemon --max-time=7300 --max-jobs=10
autostart=true
autorestart=true
stopasgroup=true
killasgroup=true
user=caddy
numprocs=3
redirect_stderr=true
stdout_logfile=/my-app/storage/logs/long-queue-workers.log
stopwaitsecs=7800
```

**Comment by wsamoht** [comment]
I found the following when looking into this back in November 2024. We encountered it on a job that completed super quick. The `completed_at` was before the `created_at` date.

| status | created_at | reserved_at | completed_at | updated_at |
|--------|------------|-------------|--------------|------------|
| pending | 1730869202.**4892** | 1730869202.4553 | 1730869202.**4858** | 1730869202.4892 |

Our fix was to use [Delayed Dispatching](https://laravel.com/docs/12.x/queues#delayed-dispatching) for the specific job affected. We are delaying 5 seconds but a second would probably work too. For us, it is just a logging job so not time sensitive.

**Comment by Parables** [comment]
Back to using Horizon again, turns out it was my fault... 

The issue: dd()

Turns out somewhere in my code, I was using `dd()`  and that caused this whole problem...

**Comment by thaifanisalla** [comment]
Having same issue (stuck on pending even with delay(1 second), and it happens randomly.

PHP: 8.3
Laravel: 10.48.29
Horizon :5.33.5


<img width="900" height="261" alt="Image" src="https://github.com/user-attachments/assets/7c78d5f9-9578-4f0c-8862-9d668aaefe2d" />

**Comment by chimit** [comment]
This happened to me on environments not explicitly mentioned in the `horizon.php`. It's not obvious but you **MUST mention every environment** your application runs in. Even empty arrays are OK. Without it on the main page on the Horizon dashboard the last section with supervisors is empty and jobs fall into "pending" constantly.

```php
'environments' => [
    'local' => [],
    'development' => [],
    'production' => [],
],
```

**Comment by pnlinh** [comment]
> Having same issue (stuck on pending even with delay(1 second), and it happens randomly.
> 
> PHP: 8.3 Laravel: 10.48.29 Horizon :5.33.5
> 
> <img alt="Image" width="900" height="261" src="https://private-user-images.githubusercontent.com/93094459/495271149-7c78d5f9-9578-4f0c-8862-9d668aaefe2d.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjAwMDI1NzcsIm5iZiI6MTc2MDAwMjI3NywicGF0aCI6Ii85MzA5NDQ1OS80OTUyNzExNDktN2M3OGQ1ZjktOTU3OC00ZjBjLTg4NjItOWQ2NjhhYWVmZTJkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTEwMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMDA5VDA5MzExN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWVjNjk5YjAxYWU5YjNiOTJjMDI1Mzg3ZmNjODdmYWUzZDEzODFhNTdlYmZlNWQ0MTM1MDljMjRjYzMyNGYzOGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.dwNuV_RDhwBRdZ9Qqn-PR5OReKd2ybkAyc_yPESW7nw">

You could check Redis log (if using queue drive is Redis). I've this issue before because have Redis connection timeout issue
